Header header
float32 range
float32 doppler_velocity
float32 bearing
float32 amplitude
uint8 msg_counter
uint8 sensor_nr
uint8 target_format_type
uint8 usage
float32 doppler_alias

